deployment/templates/mongodb-pvc.yaml

```diff
@@ -0,0 +1,13 @@
+---
+{{- if and (not .Values.development) (.Values.mongodb.persistent) }}
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: "{{ .Values.name }}-mongodb"
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 5Gi
+{{- end }}
```

Comment:
Needed?

Reply:
keeping for now because i think it might still be used if i'm reading the values.yaml right.  not sure if we can just set persistent to false there and then delete this?  or if there's more to it.